core/store/models/job_spec.go

```diff
@@ -41,13 +42,14 @@ type TaskSpecRequest struct {
 // for a given contract. It contains the Initiators, Tasks (which are the
 // individual steps to be carried out), StartAt, EndAt, and CreatedAt fields.
 type JobSpec struct {
-	ID         string      `json:"id,omitempty" gorm:"primary_key;not null"`
-	CreatedAt  time.Time   `json:"createdAt" gorm:"index"`
-	Initiators []Initiator `json:"initiators"`
-	Tasks      []TaskSpec  `json:"tasks"`
-	StartAt    null.Time   `json:"startAt" gorm:"index"`
-	EndAt      null.Time   `json:"endAt" gorm:"index"`
-	DeletedAt  null.Time   `json:"-" gorm:"index"`
+	ID         string       `json:"id,omitempty" gorm:"primary_key;not null"`
+	CreatedAt  time.Time    `json:"createdAt" gorm:"index"`
+	Initiators []Initiator  `json:"initiators"`
+	MinPayment *assets.Link `json:"minPayment" gorm:"type:varchar(255)"`
```

Comment:
Without taking a snapshot of JobSpec in migration0, this column will be added in migration0, then the alter above will fail.